src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerValidator.cs

```diff
@@ -472,11 +472,11 @@ public List<TableDesignerIssue> Run(Dac.TableDesigner designer)
         {
             var table = designer.TableViewModel;
             var errors = new List<TableDesignerIssue>();
-            if (!table.IsEdge && table.Columns.Items.Count == 0)
+            if (!table.IsEdge && table.Columns.Items.Where(c => !c.IsComputed).Count() == 0)
```

Comment:
Not sure if it's better to do the validation here or add a separate rule.

Reply:
i think this is fine